web/themes/custom/hoeringsportal/templates/layout/region--header.html.twig

```diff
@@ -0,0 +1,28 @@
+{#
+/**
+ * @file
+ * Theme override to display a region.
+ *
+ * Available variables:
+ * - content: The content for this region, typically blocks.
+ * - attributes: HTML attributes for the region div.
+ * - region: The name of the region variable as defined in the theme's
+ *   .info.yml file.
+ *
+ * @see template_preprocess_region()
+ */
+#}
+{% if content %}
```

Comment:
This should probable be changed to `{% if elements %}` (we don't use `content` elsewhere in the template) or be removed.

It seems weird to me that we don't have a wrapper element (like `<div class="header">…</div>`, say) around the content in this template, but that's just a personal opinion.

Reply:
That makes perfectly sense. Removed `content` and moved the div container from the parent template, into this template, with this commit: https://github.com/itk-dev/hoeringsportal/pull/67/commits/689e384e53c1ce1b6472f21ebd468cbc06c348c9